src/schema.php

```diff
@@ -143,6 +154,13 @@ public function replace_domain( $data ) {
 		return $this->array_value_str_replace( $source, $target, $data );
 	}
 
+	/**
+	 * Enables the feature flag for the structured data blocks.
+	 */
+	public function enable_feature_flag() {
+		\define( 'YOAST_SEO_SCHEMA_BLOCKS', true ); // phpcs:ignore WordPress.NamingConventions.PrefixAllGlobals.NonPrefixedConstantFound -- The prefix matches that of Yoast SEO, where this flag belongs.
```

Comment:
I would add a check around this to prevent errors when the constant already has been defined.
```suggestion
		if ( defined( 'YOAST_SEO_SCHEMA_BLOCKS' ) ) {
			return;
		}

		\define( 'YOAST_SEO_SCHEMA_BLOCKS', true ); // phpcs:ignore WordPress.NamingConventions.PrefixAllGlobals.NonPrefixedConstantFound -- The prefix matches that of Yoast SEO, where this flag belongs.
```

Reply:
Done